lib/plugins/base.py

```diff
@@ -0,0 +1,66 @@
+from html.parser import HTMLParser
+
+from markdown import markdown
```

Comment:
```suggestion
from markdown2 import markdown
```

Reply:
yes, but lets keep this change out of here and in a separate PR or issue where the pros and cons can be discussed properly.